bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/BallerinaValues.java

```diff
@@ -76,17 +79,51 @@
      * @param fieldValues values to be used for fields when creating the object value instance.
      * @return value of the object.
      */
+    @Deprecated
     public static ObjectValue createObjectValue(BPackage packageId, String objectTypeName, Object... fieldValues) {
+        return createObjectValue(packageId, objectTypeName, getStrand(), fieldValues);
+    }
+
+    private static ObjectValue createObjectValue(BPackage packageId, String objectTypeName, Strand currentStrand,
+                                                Object... fieldValues) {
+        //This method duplicates the createObjectValue with referencing the issue in runtime API getting strand
         ValueCreator valueCreator = ValueCreator.getValueCreator(packageId.toString());
         Object[] fields = new Object[fieldValues.length * 2];
+        Scheduler scheduler = currentStrand != null ? currentStrand.scheduler : null;
```

Comment:
can we remove individual null checks and move it to the if block below (at line 104)